key-uglifier/src/uglifier.rs

```diff
@@ -1,70 +1,55 @@
 use colored::Colorize;
 use hex::ToHex;
+use std::fmt::Write;
 
-fn param_set_default_encryption_key(key_length: u8) -> Vec<u8>
-{
-    let default_key = vec![0x62u8; key_length as usize];
-    return default_key
+fn param_set_default_encryption_key(key_length: u8) -> Vec<u8> {
+    vec![0x62u8; key_length as usize]
 }
 
-fn read_hex_nibble(c: char) -> (bool, String)
-{
-    if (c >='0') && (c <= '9'){
-        return (true, c.to_string());
-    }
-    else if (c >='A') && (c <= 'F')
-    {
-        return (true, c.to_string());
-    }
-    else if (c >='a') && (c <= 'f')
-    {
-        return (true, c.to_string());
-    }
-    else
-    {
+fn read_hex_nibble(c: char) -> (bool, String) {
+    if "0123456789ABCDEFabcdef".contains(c) {
```

Comment:
For this purpose, there is a [char::is_digit](https://doc.rust-lang.org/std/primitive.char.html#method.is_digit)